tools/clang/tools/dxcompiler/dxcpdbutils.cpp

```diff
@@ -259,6 +260,7 @@ struct DxcPdbUtils : public IDxcPdbUtils, public IDxcPixDxilDebugInfoFactory
   hlsl::DxilCompilerVersion m_VersionInfo;
   std::string m_VersionCommitSha;
   std::string m_VersionString;
+  CComPtr<IDxcCompiler3> m_pCompiler;
```

Comment:
A comment in Reset() expressing that this is not reset on purpose would help.  Otherwise some maintainer might think it was forgotten.